src/routes/users.js

```diff
@@ -171,7 +173,11 @@ exports.plugin = {
               birthCity: Joi.string(),
               socialSecurityNumber: Joi.number(),
             }),
-            contact: Joi.object().keys({ phone: phoneNumberValidation.allow('', null), address: addressValidation }),
+            contact: Joi.object().keys({
+              phone: phoneNumberValidation.allow('', null),
+              countryCode: countryCodeValidation.allow('', null),
+              address: addressValidation,
+            }).and('phone', 'countryCode'),
```

Comment:
Ici tu pourrais aussi virer ce and car dans le cas ou on envoie le phone vide, on veut plus envoyer `countryCode`. Tu peux plutot ajouter une condition sur `countryCode` : .when('contact.phone', { is: Joi.exist(), then: Joi.required(), otherwise: Joi.forbidden() }` non ?

Reply:
c'est l'inverse mais oui